packages/alfa-rules/src/sia-r68/rule.ts

```diff
@@ -97,3 +89,33 @@ function isRequiredChild(
     };
   }
 }
+
+/**
+ * Collect all descendants of node that:
+ * * are non-ignored HTML or SVG elements with a role requiring specific children; and
+ * * do not have an aria-busy ancestor
+ */
+function* visit(node: Node, device: Device): Iterable<Element> {
+  if (and(isElement, hasAttribute("aria-busy", equals("true")))(node)) {
+    return [];
```

Comment:
```suggestion
    return;
```

For generator functions, the concepts of "yielding" and "returning" are different. Whatever a generator function returns will be available as the _final_ value _after_ it is done yielding values. In this case, we only care about the yielded values of type `Element` and so might as well have the generator return `void` rather than `never[]`.